person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    isof_age? || @parent_permission
+  end
+
+  private
+
+  def isof_age?
```

Comment:
- [ ] **[OPTIONAL]** Don't you think it would be better to name this method `is_of_age?` as it is more readable that way? Or better: `of_age?`